test/cbmc/proofs/ARP/ARPAgeCache/ARPAgeCache_harness.c

```diff
@@ -1,11 +1,24 @@
 /* FreeRTOS includes. */
 #include "FreeRTOS.h"
 #include "list.h"
+#include "task.h"
 
 /* FreeRTOS+TCP includes. */
 #include "FreeRTOS_IP.h"
 #include "FreeRTOS_IP_Private.h"
 #include "FreeRTOS_ARP.h"
+#include "FreeRTOS_Routing.h"
+
+/* Provide a stub for the interface output function. This is the low
+ * level output function which is platform dependent. */
+BaseType_t xLocalNetworkInterfaceOutput( NetworkInterface_t * pxInterface,
+                                         NetworkBufferDescriptor_t * const pxNetworkBuffer,
+                                         BaseType_t bReleaseAfterSend )
+{
+    BaseType_t xReturn;
+
+    return xReturn;
```

Comment:
What about checking (asserting) that the pointer arguments are nonnull?

Reply:
Good catch! I'll add it.

Reply:
Added.